molconfgen/output.py

```diff
@@ -0,0 +1,79 @@
+# Richard Richardson wrote this ;)
+# Function to add a box to the output of
+# sampler.generate_conformers
+
+import MDAnalysis as mda
+import MDAnalysis.transformations
+import numpy as np
+
+
+# Function for finding the largest radius in
+# the trajectory
+# this will come in handy later
+def largest_r(u):
+    r = 0
+    for ts in u.trajectory:
+        ag = u.atoms
+        rnew = ag.bsphere()[0]
+        if rnew > r:
+            r = rnew
+    return r
+
+def write_pbc_trajectory(u, filename, box = None):
```

Comment:
Follow the numpy docstring format https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
Explain in particular what the `box` argument is doing.